samples/azure-graalvm-storage-samples-spring/pom.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.6.6</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-graalvm-storage-samples-spring</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>Azure Storage GraalVM Samples using Spring</name>
+  <description>Azure Storage Explorer sample</description>
```

Comment:
I wonder if, to avoid embarrassment, we set the group ID to something we can't publish to, e.g. `com.azure.samples`?

Reply:
And, if so, we should change our sample package structure to `com.azure.samples.graalvm.storageexplorer`?

Reply:
Changed the group name to `com.azure.samples` and artifact name to `azure-samples-graalvm-spring-storageexplorer` and the root package is `com.azure.samples.graalvm.spring.storageexplorer`.